### PR TITLE
Bypass fastly cleanup for Origin Cloaking

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Changes the title of [Custom VCL to bypass Fastly](https://devdocs.magento.com/cloud/cdn/fastly-vcl-bypass-to-origin.html) to "Custom VCL to bypass Fastly cache" to clarify purpose of the VCL.

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/cdn/fastly-vcl-bypass-to-origin.html

### CODE REVIEW

File: src/cloud/cdn/fastly-vcl-bypass-to-origin.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+title: Custom VCL to bypass Fastly cache
 redirect_from:
    - /cloud/configure/fastly-vcl-badreferer.html
 functional_areas:
   - Cloud
   - Setup
 ---
 
-You can create a custom VCL snippet to bypass Fastly caching and submit requests directly to the origin server, for example to determine whether site issues are caused by caching, or to troubleshoot headers. You can configure the snippet to bypass Fastly for requests from a specific IP address or URL.
+You can create a custom VCL snippet to bypass Fastly caching and submit requests directly to the origin server, for example to determine whether site issues are caused by caching, or to troubleshoot headers. You can configure the snippet to bypass caching for requests from a specific IP address or URL.
```
Comment (BarnyShergold) [inline src/cloud/cdn/fastly-vcl-bypass-to-origin.md:11]: I would suggest "Fastly caching" as this eliminates the ambiguity between Fastly cache and Magento cache

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Please also correct typo in PR title (Mae should read Magento)

**Comment by meker12** [comment]
> Please also correct typo in PR title (Mae should read Magento)

I can provide a better title, but "MAE" are my initials ;-).

File: src/cloud/cdn/fastly-vcl-bypass-to-origin.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ---
 group: cloud-guide
-title: Custom VCL to bypass Fastly
+title: Custom VCL to bypass Fastly cache
```
Comment (meker12) [inline src/cloud/cdn/fastly-vcl-bypass-to-origin.md:3]:
```suggestion
title: Custom VCL to bypass Fastly caching
```

File: src/cloud/cdn/fastly-vcl-bypass-to-origin.md (hunk 1 above)
Comment (meker12) [inline src/cloud/cdn/fastly-vcl-bypass-to-origin.md:11]:
```suggestion
You can create a custom VCL snippet to bypass Fastly caching and submit requests directly to the origin server, for example to determine whether site issues are caused by caching, or to troubleshoot headers. You can configure the snippet to bypass Fastly caching for requests from a specific IP address or URL.
```

**Review by BarnyShergold — APPROVED** [review]
Apologies about the title - thought it was a typo of Magento :)

**Review by jeff-matthews — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**